aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -156,6 +156,21 @@ Tensor& addr_out(Tensor &result, const Tensor& self, const Tensor& vec1, const T
   return at::_addr_out(result, self, vec1, vec2, beta, alpha);
 }
 
+Tensor& ger_out(Tensor &result, const Tensor& self, const Tensor& vec2) {
+  check_1d(self, "self", "ger");
+  check_1d(vec2, "vec2", "ger");
+  if (result.numel() == 0) {
+    result.resize_({ self.size(0), vec2.size(0) });
+  }
+  return at::_addr_out(result, result, self, vec2, Scalar(0), Scalar(1));
```

Comment:
This doesn't seem right: `at::_addr_out(result, result, self, vec2, ...)` if I understand it correctly does:
result = (result + outer_product(self, vec2)). What we really want here is `result = outer_product(self, vec2)`, right?

Reply:
Nvm, I see. The beta=0 makes it so that the addition is irrelevant